ClientLibrary/lib/src/main/java/com/microsoft/projectoxford/face/FaceServiceClient.java

```diff
@@ -329,15 +378,47 @@ public String toString() {
     Person getPerson(String personGroupId, UUID personId) throws ClientException, IOException;
 
     /**
-     * Gets all persons inside a person group.
+     * Gets 1000 persons inside a person group.
      * @param personGroupId The person group id.
      * @return The person entity array.
      * @throws ClientException
      * @throws IOException
+     * @deprecated use {@link #listPersons(String)} l} instead.
      */
+    @Deprecated
     Person[] getPersons(String personGroupId) throws ClientException, IOException;
 
     /**
+     * List the fist "top" of persons whose Id is lager than "start".
+     * @param personGroupId The person group id.
+     * @param start The persons Id bar, list persons whose Id is lager than "start.
+     * @param top The number of persons to list.
+     * @return The person entity array.
+     * @throws ClientException
+     * @throws IOException
+     */
+    Person[] listPersons(String personGroupId, String start, int top) throws ClientException, IOException;
```

Comment:
missing the function with only "top" parameter. Please check listPersongroups method as well.